controllers/list/submissions/PKPSubmissionsListHandler.inc.php

```diff
@@ -139,11 +121,12 @@ public function getConfig() {
 			'yes' => __('common.yes'),
 			'no' => __('common.no'),
 			'deleting' => __('common.deleting'),
+			'currentStage' => __('submission.list.currentStage'),
 			'confirmDelete' => __('submission.list.confirmDelete'),
 			'responseDue' => __('submission.list.responseDue'),
 			'reviewDue' => __('submission.list.reviewDue'),
-			'filter' => __('submission.list.filter'),
-			'filterRemove' => __('submission.list.filterRemove'),
+			'filter' => __('common.filter'),
+			'filterRemove' => __('common.filterRemove'),
```

Comment:
this locale key contain a variable, that is not provided here, or?

Reply:
The Vue.js components include a `__()` mixin which can compile translations within Vue components. That way we can update strings based on the data loaded into the component:

https://github.com/pkp/ui-library/blob/master/src/mixins/global.js#L14-L45